### ISSUE TITLE
NTR: Myelin basic protein high in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8462]
New term request Myelin basic protein high in CSF (segal_110418211139): Cerebrospinal fluid  UMLS: 4316873

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000205
- label: Elevated CSF myelin basic protein concentration
- parents: [Abnormal CSF protein concentration (HP:0025456)](https://hpo.jax.org/app/browse/term/HP:0025456)
- pmid: PMID:12465458
- def: The concentration of myelin basic protein in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: Radioimmunoassay for myelin basic protein in cerebrospinal fluid is commonly used as a biochemical marker of demyelination in multiple sclerosis patients.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8462
- synonyms: Myelin basic protein high in CSF(NONE)
